Samples/resource-owner-password-credential/Angular2/Angualar2-Client-ROPC/app/authorize/authorize-component.html

```diff
@@ -0,0 +1,81 @@
+﻿<ul *ngIf="!isLoggedin" class="nav navbar-nav navbar-right">
+    <li><button class="btn btn-link" (click)="mopen()">Login</button></li>
+</ul>
+<ul *ngIf="isLoggedin" class="nav navbar-nav navbar-right">
+    <li><button class="btn btn-link">Welcome {{userDetails.userName}}</button></li>
+    <li><button class="btn btn-link" (click)="logout()">Logout</button></li>
+</ul>
+<modal #myModal id="login-modal" tabindex="-1" role="dialog" style="display: none;">
+    <modal-header style="text-align:center">
+        <img class="img-circle" width="100" id="img_logo" src="http://bootsnipp.com/img/logo.jpg">
+    </modal-header>
+    <modal-body>
+        <form id="login-form" name="Loginform" [hidden]="!loginBool">
```

Comment:
actually its required if its a group of element eg. label and input but its not i am using placeholder instead ... (It is recommended to have invisible labels with sr-only class for screen readers but i dont think it's necessary as this design is very simple and users will have to implement their own design but let me know what you think)